### ISSUE TITLE
Docs for install cuda 11.0 and cudnn 8

### ISSUE THREAD

**Issue by lminer** [tensorflow/tensorflow#45308]
Now that tensorflow 2.4.0 is almost out with cuda 11.0 and cudnn 8 support, it would be nice to have the instructions updated for GPU installation.

**Comment by Harsh188** [comment]
Hi, I'm willing to work on this issue. However, there are many issues at the moment with RTX cards running on CUDA 11.0. Some suggest that 11.1 works while 11.0 doesn't. Personally, I was facing issues even while using the TF docker container until I was facing issues even while using the TF docker container. I stumbled across an article by [lambda stack](https://lambdalabs.com/blog/install-tensorflow-and-pytorch-on-rtx-30-series/) which allowed me to use TF 2.3 on my new RTX 3080. 

So I suggest waiting on this issue for a little while longer until the issues have been sorted out. In the meantime, I'll start up a pull request since I think we should refactor the [GPU support](https://www.tensorflow.org/install/gpu) page.

**Comment by sanjoy** [comment]
> Some suggest that 11.1 works while 11.0 doesn't.

@nluehr Is this true?  This could be a reason to move to 11.1 or 11.2 sooner rather than later.

CC @pkanwar23

**Comment by pkanwar23** [comment]
FYI. GPU instructions have been updated. There might be minor omissions, which I'll fix soon.

**Comment by cycks** [comment]
I was trying to reproduce the instruction [here](https://www.tensorflow.org/install/gpu) I was doing it in version20.04. 
I started by running `ubuntu-drivers devices` to find the recommended drivers for my version. The recommended version is `nvidia-driver-455 - third-party free recommended`.

Using the instructions [here](https://www.tensorflow.org/install/gpu#ubuntu_1804_cuda_110).
I replaced 1804 with 2004 as shown in the lines below.
`wget https://developer.download.nvidia.com/compute/cuda/repos/ubuntu2004/x86_64/cuda-ubuntu2004.pin`
`sudo mv cuda-ubuntu1804.pin /etc/apt/preferences.d/cuda-repository-pin-600`
`sudo apt-key adv --fetch-keys https://developer.download.nvidia.com/compute/cuda/repos/ubuntu2004/x86_64/7fa2af80.pub`
`sudo add-apt-repository "deb https://developer.download.nvidia.com/compute/cuda/repos/ubuntu2004/x86_64/ /"`
`sudo apt-get update`
`wget http://developer.download.nvidia.com/compute/machine-learning/repos/ubuntu2004/x86_64/nvidia-machine-learning-repo-ubuntu2004_1.0.0-1_amd64.deb`
`sudo apt install ./nvidia-machine-learning-repo-ubuntu2004_1.0.0-1_amd64.deb`
`sudo apt-get update`

`# Install NVIDIA driver`
`sudo apt-get install --no-install-recommends nvidia-driver-455`
`sudo reboot`
 Reboot. Check that GPUs are visible using the command: nvidia-smi
However, I was stuck when I tried to replace the following line because ubuntu 2004 has no `libnvinfer7_7.1.3-1+cuda11.0_amd64.deb` counterpart that is available in 18004.
[check here](https://developer.download.nvidia.com/compute/machine-learning/repos/ubuntu2004/x86_64/). 
So my question is, is their a package i need to install in version 2004 that will replace the libnvinfer7_7.1.3-1+cuda11.0_amd64.deb` version in ubuntu 18?

**Comment by Saduf2019** [comment]
Is this still an issue.

**Comment by Saduf2019** [comment]
this has been fixed, please refer [to link](https://www.tensorflow.org/install/gpu), moving this to closed status.
